e2e_tests/tests/deploy/test_local.py

```diff
@@ -103,7 +103,29 @@ def resource_manager(
 def test_cluster_down() -> None:
     name = "test_cluster_down"
 
-    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu"]):
+    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu", "enterprise-edition"]):
```

Comment:
How is bringing down an EE cluster different from bringing down an OSS one?

Names appear to be twiddled in `test_cluster_down` and `test_ee_cluster_down` relative to the passed flags, but I don't see anything in the test that tests the flag.